pkg/trace/api/profiles.go

```diff
@@ -76,6 +105,56 @@ func newProfileProxy(transport http.RoundTripper, target *url.URL, apiKey, tags
 	return &httputil.ReverseProxy{
 		Director:  director,
 		ErrorLog:  stdlog.New(logger, "profiling.Proxy: ", 0),
-		Transport: transport,
+		Transport: &multiTransport{transport, targets, keys},
+	}
+}
+
+// multiTransport sends HTTP requests to multiple targets using an
+// underlying http.RoundTripper. API keys are set separately for each target.
+// When multiple endpoints are in use the response from the main endpoint
+// is proxied back to the client, while for all aditional endpoints the
+// response is discarded. There is no de-duplication done between endpoint
+// hosts or api keys.
```

Comment:
Is the de-duplication comment relevant?

Reply:
I added it because I noticed we dedupe in other places: https://github.com/DataDog/datadog-agent/blob/4abf72dd4c8246bcb096f54a68f66f1839d7eff1/pkg/config/config.go#L1073

I'm fine with removing it if you think it's not relevant, or we could implement deduping 🤷‍♂️